src/synthesizer/particle/blackholes.py

```diff
@@ -396,9 +396,9 @@ def _prepare_sed_args(
             for prop in props
         ]
 
-        # For black holes mass is a grid parameter but we still need to
-        # multiply by mass in the extensions so just multiply by 1
-        mass = np.ones(npart, dtype=np.float64)
+        # For black holes the grid Sed are normalised to 1.0 so we need to
+        # scale by the bolometric luminosity.
+        bol_lum = self.bolometric_luminosity.value
```

Comment:
You also need to do this in `parametric/blackhole.py` in the same functions at lines 198 and 335.

Reply:
Done